index.bs

```diff
@@ -1726,14 +1741,14 @@ When this operation is invoked, the authenticator must perform the following pro
         :  a global [=signature counter=]
         ::  Use the global [=signature counter=]'s actual value when generating 
               [=authenticator data=].
+        : a per credential [=signature counter=]
+        :: allocate the counter, associate it with the new credential, and initialize the counter value as zero.
```

Comment:
@rlin1 This collided a bit with #539. Did I merge it reasonably?

Reply:
Not to speak for Rolf, but it looks right to me.

Reply:
this looks fine to me! we should merge this.